Formula/heimdal.rb

```diff
@@ -14,6 +13,11 @@ class Heimdal < Formula
 
   keg_only :shadowed_by_macos, "macOS provides Kerberos"
 
+  depends_on "berkeley-db"
+  depends_on "bison"
+  depends_on "flex"
+  depends_on "lmdb"
+  depends_on "openldap"
```

Comment:
Are those all runtime dependencies?

Reply:
I'm honestly not sure haha

Reply:
Okay so I tested `brew linkage --test heimdal` and it looks like bison can be marked as a compile-time dependency